code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm

```diff
@@ -865,10 +898,56 @@ datum/reagent/medicine/kelotane/on_mob_life(mob/living/M)
 	return
 
 datum/reagent/medicine/kelotane/overdose_process(mob/living/M)
-	M.adjustFireLoss(4*REM)
+	M.adjustFireLoss(3*REM)
+	metabolization_rate = 3 * REAGENTS_METABOLISM
 	..()
 	return
 
+datum/reagent/medicine/bromelain
+	name = "Bromelain"
+	id = "bromelain"
+	description = "Effective at healing severe brusing and very efficient but acts somewhat slowly."
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+	overdose_threshold = 30
+	metabolization_rate = 0.5 * REAGENTS_METABOLISM
+
+datum/reagent/medicine/bromelain/on_mob_life(mob/living/M)
+	M.adjustBruteLoss(-2*REM)
+	if(M.getBruteLoss() < 30)
```

Comment:
Shouldn't you add an else here to reset metabolization rate?
If someone dips below 30 brute then goes back up then the metabolization stays at 0.4 instead of 0.2.
Unless this is intended behavior.